shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -753,6 +810,46 @@ enum RewriteTableTy {
     Peers,
 }
 
+fn is_prepared_message(message: &mut Message) -> Option<(Vec<u8>, PreparedMetadata)> {
+    if let Some(Frame::Cassandra(CassandraFrame {
+        operation: CassandraOperation::Result(CassandraResult::Prepared(prepared)),
+        ..
+    })) = message.frame()
+    {
+        return Some((
+            prepared.id.clone().into_bytes().unwrap(),
+            prepared.metadata.clone(),
+        ));
+    }
+
+    None
+}
+
+fn is_execute_message(message: &mut Message) -> Option<(&BodyReqExecuteOwned, &Version)> {
```

Comment:
I think `is_` prefix makes sense for `bool`s because it describes what the bool represents, but in cases where we are returning message contents a `get_` prefix would be better.